reference_system/include/reference_system/system/timing/default.hpp

```diff
@@ -23,13 +23,13 @@ namespace timing
 struct Default
 {
   static constexpr std::chrono::nanoseconds SENSOR_CYCLE_TIME =
-    std::chrono::milliseconds(200);
+    std::chrono::milliseconds(100);
```

Comment:
@elfenpiff from our dialog this morning with MP I went ahead and reduced the default times. we can add more timing configs later if we want